microsoft-edge/webview2/how-to/winrt-js-conversion.md

```diff
@@ -36,6 +36,10 @@ The WebView2 WinRT JS Projection tool (**wv2winrt**) converts from WinRT to Java
 | Class static member | JavaScript object property | See below. |
 | Class constructor | JavaScript constructor and function | See below. |
 
+A couple notes for passing JavaScript objects to host objects:
+*  If JavaScript Date objects need to be passed to host objects as `VT_DATE`, the host objects property `shouldSerializeDates` should be set to true. By default, it is passed to host as string using `JSON.stringify`.
+*  If JavaScript typed arrays need to be passed to host objects as array, the host objects property `shouldPassTypedArraysAsArrays` should be set to true. By default, it is passed to host as `IDispatch`.
```

Comment:
```suggestion
*  If JavaScript typed arrays need to be passed to host objects as `array`, set the host object's `shouldPassTypedArraysAsArrays` property to `true`. By default, typed arrays are passed to the host as `IDispatch`.
```
* should - corp-level style guide re: "should": https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/s/should-vs-must
* it - check whether "it" means typed arrays.  prefer being specific instead of "it".